pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1791,6 +1792,26 @@ Question.Decimal:
           type: array
           items:
             $ref: '#/Answer.Decimal'
+Question.Equation:
+  allOf:
+    - $ref: '#/Question'
+    - type: object
+      required:
+        - answers
+      properties:
+        expression:
+          type: string
+          description: the expression that must be computed.
```

Comment:
Why does the client need knowledge of the equation? As far as I understand, there shouldn't be anything for a client to do aside from present the value computed by the server.

Reply:
I supposed that we might want to display the formula to let the client know what's going on and how we get the number.

Reply:
The formula will be stored in the DB anyway, but ok, I'll remove it from the definition